forge/src/multi_runner.rs

```diff
@@ -125,6 +133,8 @@ impl MultiContractRunner {
         // the db backend that serves all the data, each contract gets its own instance
         let db = Backend::spawn(self.fork.take());
 
+        let error_reported = Arc::new(AtomicBool::new(false));
+
```

Comment:
I learned recently that we don't need the Arc here, because this var isn't moved.